.github/workflows/test-linter-build-push.yml

```diff
@@ -32,16 +32,13 @@ jobs:
       - name: Install dependencies
         # the reason why hardcoded the dependencies instead of running "make .install" is that make .install throws an error related to go mod
         run: |
-          go get -modfile=go.tools.mod github.com/golang/protobuf/protoc-gen-go google.golang.org/grpc/cmd/protoc-gen-go-grpc && \
-          go get google.golang.org/grpc/cmd/protoc-gen-go-grpc && \
+          go install -modfile=go.tools.mod github.com/golang/protobuf/protoc-gen-go google.golang.org/grpc/cmd/protoc-gen-go-grpc && \
           go get github.com/ghodss/yaml@v1.0.0 && \
-          go install google.golang.org/grpc/cmd/protoc-gen-go-grpc && \
-          go get github.com/libgit2/git2go/v33 && \
+          go install github.com/libgit2/git2go/v33 && \
           go install \
             github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway \
             github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2 \
             google.golang.org/protobuf/cmd/protoc-gen-go \
-            google.golang.org/grpc/cmd/protoc-gen-go-grpc \
             github.com/grpc-ecosystem/grpc-gateway/v2/internal/descriptor
```

Comment:
we need to run this locally anyway, right? Please try to move this to the makefile. Or if that is not possible, explain more why (line 33)

Reply:
Yes, I think that makes more sense